ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -268,7 +252,7 @@ class LeftNavMenuViewController: UIViewController {
         contentView.addSubview(accountView)
         contentView.addSubview(menuListView)
 
-        NSLayoutConstraint.activate([accountView.heightAnchor.constraint(equalToConstant: 100),
+        NSLayoutConstraint.activate([accountView.heightAnchor.constraint(equalToConstant: 84),
```

Comment:
I believe the UIView returned by the personaview will already have its height set, which makes this not needed.

Reply:
Without setting priority layouts, the accountView (aka persona) ends up getting stretched and pulled to the bottom, so I left it there

Reply:
menu list height should be more dynamically stretched though. is there a priority difference? https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526946-priority